gammapy/maps/tests/test_hpx.py

```diff
@@ -838,6 +838,21 @@ def test_hpx_geom_region_mask():
     solid_angle = (mask.data * geom.solid_angle()).sum()
     assert_allclose(solid_angle, 2 * np.pi * (1 - np.cos(3 * u.deg)) * u.sr, rtol=0.01)
 
-
-
+def test_hpx_geom_separation():
+    geom = HpxGeom.create(skydir=(0,0), binsz=0.1, frame="galactic", nest=True)
```

Comment:
Minor in case of the all sky geometry providing the `skydir` has no effect, just remove...